src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
```

Comment:
The opening `<html>` tag and its attributes should be formatted properly with each attribute starting on a new line with a 2-space indentation relative to the tag.